kit/DummyLibreOfficeKit.cpp

```diff
@@ -38,115 +38,115 @@ struct LibLibreOffice_Impl : public _LibreOfficeKit
     LibLibreOffice_Impl();
 };
 
-static LibLibreOffice_Impl *gImpl = nullptr;
-static std::weak_ptr< LibreOfficeKitClass > gOfficeClass;
-static std::weak_ptr< LibreOfficeKitDocumentClass > gDocumentClass;
+static LibLibreOffice_Impl *impl = nullptr;
```

Comment:
Nit: this is modeled after core.git desktop/source/lib/init.cxx, so with this rename you introduce some unwanted differences.